pkg/cmd/pulumi/about.go

```diff
@@ -0,0 +1,923 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Nit: Delete extra line

```suggestion
// Copyright 2016-2021, Pulumi Corporation.
//
// Licensed under the Apache License, Version 2.0 (the "License");
```